Content.Client/UserInterface/Systems/Character/CharacterUIController.cs

```diff
@@ -94,6 +94,29 @@ public void LoadButton()
     private void DeactivateButton() => CharacterButton!.Pressed = false;
     private void ActivateButton() => CharacterButton!.Pressed = true;
 
+    private const int MaxObjectiveLineLength = 38;
+    private string WordWrap(string inputText, int lineLength)
```

Comment:
Word wrapping should split text based on how wide it is, not on how many (variable-width) characters should be in a line. You don't want to do this by hand, either; there are controls which already have word wrapping functionality, which you should use instead - either `RichTextEntry` or `TextEdit` (with `Editable=false`)

Reply:
Thanks for pointing out RichTextEntry. Makes this much easier.